lib/membership/update.js

```diff
@@ -24,15 +24,29 @@ var Member = require('./member.js');
 var uuid = require('node-uuid');
 var util = require('util');
 
-function Update(address, incarnationNumber, status, localMember) {
-    this.address = address;
-    this.incarnationNumber = incarnationNumber;
-    this.status = status;
+/**
+ * Create a new Update
+ * @param {IMember} subject the subject of the update.
+ * @param {IMember} [source] the source of the update.
+ * @constructor
+ */
+function Update(subject, source) {
+    if (!(this instanceof Update)) {
```

Comment:
This branch is new in this PR, is it used or shall we remove it since you also indicated via `@constructor` that it should be invoked with `new`. Especially since the `Update` is probably not called as a function in the rest of the code.